cloud/scope/powervs_cluster.go

```diff
@@ -2498,6 +2498,10 @@ func (s *PowerVSClusterScope) DeleteDHCPServer() error {
 		s.Info("Skipping DHP server deletion as resource is not created by controller")
 		return nil
 	}
+	if s.isResourceCreatedByController(infrav1beta2.ResourceTypeServiceInstance) {
+		s.Info("Skipping DHCP server deletion as PowerVS service instance is created by controller, will directly delete the PowerVS service instance")
```

Comment:
I think user may not know PowerVS service instance delete will also delete DHCP server and may get confused after seeing work Skipping, I think its better to either use V(3) or update the log message.

Reply:
Updated the log, let's have this log IMO, since its skipping a major action in delete flow.